### DISCUSSION TITLE
sdxl-koala for faster image generation

### DISCUSSION THREAD

**Discussion by austinamorusoyardstick** [lllyasviel/Fooocus#2385] [Ideas]
https://huggingface.co/spaces/etri-vilab/KOALA

https://www.livescience.com/technology/artificial-intelligence/new-ai-image-generator-koala-is-8-times-faster-than-openais-best-tool-and-can-run-on-cheap-computers

https://github.com/youngwanLEE/sdxl-koala

**Comment by mashb1t** [comment]
Is this a feature request to add Koala?
FYI You can already set up a preset for models compatible with the standard SDXL structure on your own. Have you tested it already?

**Comment by austinamorusoyardstick** [comment]
I wasn't able to find the model file, I was being lazy saying make it work. lol
